src/components/manage/Blocks/Listing/withQuerystringResults.jsx

```diff
@@ -23,7 +23,7 @@ export default function withQuerystringResults(WrappedComponent) {
     } = props;
     const { settings } = config;
     const querystring = data.querystring || data; // For backwards compat with data saved before Blocks schema. Note, this is also how the Search block passes data to ListingBody
-
+    const subrequestID = content.UID + '-' + id;
```

Comment:
I was a bit worried in case there is ever content without a UID -- but in that case we should end up with "undefined-someid" which is no worse than before.